src/timepicker-select/timepicker-select.component.ts

```diff
@@ -27,6 +28,7 @@ import { NG_VALUE_ACCESSOR } from "@angular/forms";
 				<ng-content></ng-content>
 			</select>
 			<ibm-icon-chevron-down16 *ngIf="!skeleton" class="bx--select__arrow"></ibm-icon-chevron-down16>
```

Comment:
Does changing to the directive form `<svg ibmIconChevronDown16></svg>` improve things at all?

Reply:
It doesn't change anything

Reply:
Did you try applying the styles from https://github.com/carbon-design-system/carbon/pull/5672/files after changing to the directive?

Reply:
I haven't! I will try that out

Reply:
We should switch to the directive form anyway 👍